web/ui/src/common/miscUtils.js

```diff
@@ -208,6 +208,67 @@
                 return childCount;
             },
 
+            humanizeDuration: function(msecs) {
+                // converts millisecons to a time duration such as 4h45m20s
+
+                if (msecs === 0) return "0";
+
+                var humanized = "";
+                var ttoken = ["w","d","h","m","s","ms"];
```

Comment:
try moving `tmultiplier` up a line, and changing `ttoken` to: `var ttoken = Object.keys(tmultiplier);`